config/config/sc-config.yaml

```diff
@@ -100,6 +100,9 @@ falco:
       - macro: user_known_contact_k8s_api_server_activities
         append: true
         condition: or (container.image.repository in (ghcr.io/dexidp/dex))
+    opensearch.yaml: |-
+      - macro: user_privileged_containers
+        condition: container.image.repository=docker.io/elastisys/curl-jq and container.image.tag=latest and k8s.pod.name startswith opensearch-master-
```

Comment:
Do we need the part about `latest` tag?

Reply:
I wanted to separate it from the `cronjob` tag. but maybe it can be removed, not sure if we assign tags based on purpose

Reply:
Ah, I thought that was a different image not just a tag.
But with the check for opensearch I think it is specific enough. I just don't want the filter to stop working because we decided to start specifying the curl-jq image tag (which we should probably do).

Reply:
I removed the tag